### PR TITLE
Add Prtinfo message type allows messages go into print file.

### PR DESCRIPTION
**Pull request by qilicun**
See https://github.com/OPM/opm-common/pull/135

### CODE REVIEW

**Comment by atgeirr** [comment]
Please see my comment to OPM/opm-common#135, basically is it feasible to get tagging in MessageContainer to avoid this workaround?

**Comment by joakim-hove** [comment]
> Please see my comment to OPM/opm-common#135, basically is it feasible to get tagging in MessageContainer to avoid this workaround?

Yes - that would be my [suggestion] (https://github.com/OPM/opm-common/pull/135#issuecomment-224412548).

**Comment by atgeirr** [comment]
> Yes - that would be my suggestion.

That would let us avoid this workaround, which would be better. Do you agree, @qilicun?

**Comment by qilicun** [comment]
@atgeirr 
It seems that we agree on this https://github.com/OPM/opm-common/pull/135#issuecomment-224412548. I still think that introducing a new message type could be a good solution for **those messages** that should only go into print file. I will update the two PR according to the comments.

**Comment by atgeirr** [comment]
OK, let's first get the extra message type. I think it should be named Notification or something other than PrtInfo though. Also: its ordering in the enum definitions (here and in opm-common) should reflect the expectation that this is "more chatty than Info", so it should go in between Debug and Info.

File: opm/parser/eclipse/Parser/MessageContainer.cpp [hunk 1]
```diff
@@ -119,6 +119,21 @@ namespace Opm {
         m_messages.emplace_back( Message { Message::Problem, msg, {} } );
     }
 
+
+    void MessageContainer::note( const std::string& msg,
+                                    const std::string& filename,
```
Comment (atgeirr) [inline opm/parser/eclipse/Parser/MessageContainer.cpp]: Inconsistent indentation.

**Comment by atgeirr** [comment]
Looks fine, minor nitpick noted above. Otherwise ready to merge.

**Pull request merged**